2294/3/helpers/azure.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'scraper.rb'
+require_relative 'form_comments.rb'
+require 'net/https'
+require 'uri'
+require 'json'
+
+# Take messages from sorted by onliner's rating array
+class PackCreator
+  def initialize
```

Comment:
It seems to me that this class should be part of something else (`Analyze` I guess)